### PR TITLE
Update admin

### PR DESCRIPTION
**Pull request by yutakahoulette**

### CODE REVIEW

File: app/assets/stylesheets/super_admins/dashboard/index.css.scss [hunk 1]
```diff
@@ -1,2 +0,0 @@
-@import 'mixins';
```
Comment (yutakahoulette) [inline app/assets/stylesheets/super_admins/dashboard/index.css.scss]: unused

File: app/assets/stylesheets/super_admins/map/page.css.scss [hunk 2]
```diff
@@ -1,28 +0,0 @@
-@import 'mixins';
```
Comment (yutakahoulette) [inline app/assets/stylesheets/super_admins/map/page.css.scss]: unused

File: app/assets/stylesheets/super_admins/search/page.css.scss [hunk 3]
```diff
@@ -1,2 +0,0 @@
-@import 'mixins';
```
Comment (yutakahoulette) [inline app/assets/stylesheets/super_admins/search/page.css.scss]: unused...

File: app/controllers/super_admins_controller.rb [hunk 4]
```diff
@@ -1,34 +1,26 @@
 class SuperAdminsController < ApplicationController
+  layout "layouts/page"
```
Comment (yutakahoulette) [inline app/controllers/super_admins_controller.rb]: use the new layout for all of the views in this controller

File: app/views/super_admins/_maps.html.erb [hunk 5]
```diff
@@ -1,43 +0,0 @@
-<% content_for(:dont_track_ga) {'true'} %>
-
```
Comment (yutakahoulette) [inline app/views/super_admins/_maps.html.erb]: unused...

File: app/views/super_admins/_tabs.html.erb [hunk 6]
```diff
@@ -1,15 +0,0 @@
-<!-- partial super_admins/tabs -->
-
-<div class='pageTabs'>
-  <a href='/admin/fullcontact' class='<%= active == :fullcontact ? 'is-active' : ''%>'>
```
Comment (yutakahoulette) [inline app/views/super_admins/_tabs.html.erb]: unused...

File: app/views/super_admins/fullcontact.html.erb [hunk 7]
```diff
@@ -1,27 +0,0 @@
-
-<% content_for(:dont_track_ga) {'true'} %>
```
Comment (yutakahoulette) [inline app/views/super_admins/fullcontact.html.erb]: unused...

File: app/views/super_admins/index.html.erb [hunk 8]
```diff
@@ -0,0 +1,12 @@
+<%= content_for :javascripts do %>
+  <%= javascript_include_tag '/client/js/super-admin/page' %>
+<% end %>
+
+<% content_for :stylesheets do %>
+  <%= stylesheet_link_tag '/client/css/dashboard/page' %>
```
Comment (yutakahoulette) [inline app/views/super_admins/index.html.erb]: i actually don't need to use the dashboard styles here, so will remove this style link

File: app/views/super_admins/search_nonprofits.html.erb [hunk 9]
```diff
@@ -1,144 +0,0 @@
-<%= content_for :javascripts do %>
-	<%= javascript_include_tag '/client/js/super_admins/search/page' %>
```
Comment (yutakahoulette) [inline app/views/super_admins/search_nonprofits.html.erb]: unused...

File: app/views/super_admins/search_nonprofits.rabl [hunk 10]
```diff
@@ -1,14 +0,0 @@
-object false
-
```
Comment (yutakahoulette) [inline app/views/super_admins/search_nonprofits.rabl]: definitely not used...

File: app/views/super_admins/search_profiles.html.erb [hunk 11]
```diff
@@ -1,91 +0,0 @@
-<%= content_for :javascripts do %>
-	<%= javascript_include_tag '/client/js/super_admins/search/page' %>
```
Comment (yutakahoulette) [inline app/views/super_admins/search_profiles.html.erb]: unused...

File: client/css/global/colors.css [hunk 12]
```diff
@@ -12,7 +12,7 @@
   --blue: #0176f5;
   --blue-light: #78b6ff;  
   --blue-lighter: #d2e6ff;
-  --green: #91d349;
```
Comment (yutakahoulette) [inline client/css/global/colors.css]: make the green a little darker for better legibility

File: client/css/global/dashboard.css [hunk 13]
```diff
@@ -9,7 +9,7 @@
 }
 
 [data-ff-dashboard-main-panel] { 
-  background: var(--grey-1) 
+  background: color(black l(99.4%));
```
Comment (yutakahoulette) [inline client/css/global/dashboard.css]: i should go back and change this to a variable

File: client/css/global/typography.css [hunk 14]
```diff
@@ -19,10 +19,10 @@
 
 body {
   font-family: 
-    'OpenSans',
     -apple-system,
```
Comment (yutakahoulette) [inline client/css/global/typography.css]: i find these system fonts to actually be much more legible and nicer than Open Sans... so in this font-stack, Open Sans is sort of a fallback for OSs that don't have these system fonts.

File: client/js/components/get-valid-data.js [hunk 15]
```diff
@@ -0,0 +1,10 @@
+const flyd = require('flimflam/flyd')
+const request = require("../common/request")
```
Comment (yutakahoulette) [inline client/js/components/get-valid-data.js]: found myself rewriting this function a lot so made it into a component. but maybe it should go into /common instead.

File: client/js/super-admin/nonprofits-table.js [hunk 16]
```diff
@@ -0,0 +1,138 @@
+const R = require('ramda')
```
Comment (yutakahoulette) [inline client/js/super-admin/nonprofits-table.js]: this is the meat of the PR

File: client/js/super_admins/fullcontact/index.es6 [hunk 17]
```diff
@@ -1,56 +0,0 @@
-const render = require('ff-core/render')
-const h = require('snabbdom/h')
```
Comment (yutakahoulette) [inline client/js/super_admins/fullcontact/index.es6]: we could add this back in if wendy of roderick are missing it

File: package.json [hunk 18]
```diff
@@ -6,8 +6,8 @@
     "url": "https://github.com/CommitChange/commitchange.com"
   },
   "scripts": {
-    "watch-js": "node scripts/watch-js.js",
-    "build-js": "node scripts/build-js.js",
+    "watch-js": "node script/watch-js.js",
```
Comment (yutakahoulette) [inline package.json]: fix links to the scripts

File: app/views/super_admins/index.html.erb (hunk 8 above)
Comment (yutakahoulette) [inline app/views/super_admins/index.html.erb]: resolved

File: client/css/global/dashboard.css (hunk 13 above)
Comment (yutakahoulette) [inline client/css/global/dashboard.css]: resolved

File: client/js/components/get-valid-data.js (hunk 15 above)
Comment (yutakahoulette) [inline client/js/components/get-valid-data.js]: resolved

**Comment by yutakahoulette** [comment]
closes https://github.com/CommitChange/commitchange.com/issues/1970

File: app/controllers/super_admins_controller.rb [hunk 19]
```diff
@@ -1,34 +1,26 @@
...
+  def index 
+  end
+
+  def search_nonprofits
+    render json: QueryNonprofits.for_admin(params)
+  end
 
-	def metrics
-		render json: {
+  def search_profiles
```
Comment (jayrbolton) [inline app/controllers/super_admins_controller.rb]: Can we nuke this controller action?

File: client/js/components/get-valid-data.js [hunk 20]
```diff
@@ -0,0 +1,10 @@
+const flyd = require('flimflam/flyd')
```
Comment (jayrbolton) [inline client/js/components/get-valid-data.js]: Probably remove from one of these two directories

File: config/routes.rb [hunk 21]
```diff
@@ -189,12 +189,11 @@
...
 	# Super admin
-	match '/admin' => 'super_admins#search_nonprofits', :as => 'admin'
-	match '/admin/search_nonprofits' => 'super_admins#search_nonprofits'
-  match '/admin/search_profiles' => 'super_admins#search_profiles'
-	match '/admin/metrics' => 'super_admins#metrics'
-  match '/admin/fullcontact' => 'super_admins#fullcontact'
-  post '/admin/fullcontact_lookup' => 'super_admins#fullcontact_lookup'
+  match '/admin' => 'super_admins#index', :as => 'admin'
+  match '/admin/search-nonprofits' => 'super_admins#search_nonprofits'
+  match '/admin/search-profiles' => 'super_admins#search_profiles'
```
Comment (jayrbolton) [inline config/routes.rb]: can we remove these routes:
/admin/search-profiles
/admin/metrics
/admin/fullcontact_lookup

File: public/client/css/dashboard/page.css [hunk 22]
```diff
@@ -0,0 +1,2 @@
...hboard-main-panel]{width:100%}[data-ff-dashboard-left-panel]{left:0}[data-ff-dashboard-right-panel]{right:0}.bg-grey-1{background:#fafafa}.bg-grey-2{background:#f5f5f5}.bg-grey-3{background:#ebebeb}.bg-grey-4{background:#dbdbdb}.bg-grey-5{background:#a6a6a6}.bg-scrim{background:rgba(0,0,0,.5)}.bg-black{background:#2f2f2f}.bg-blue{background:#0176f5}.bg-blue-light{background:#78b6ff}.bg-blue-lighter{background:#d2e6ff}.bg-red{background:#ff0f0f}.bg-green{background:#82c13e}.color-white{color:#fff}.color-grey{color:gray}.color-black{color:#2f2f2f}.color-red{color:#ff0f0f}.color-green{color:#82c13e}.color-blue{color:#0176f5}.border-color-grey{border-color:#dbdbdb}.border-color-red{border-color:#ff0f0f}.border-color-green{border-color:#82c13e}.border-color-blue{border-color:#78b6ff}[data-ff-dashboard-header]{background:#fff;padding:8px 16px;padding:.5rem 1rem;border-bottom:2px solid #dbdbdb}[data-ff-dashboard-main-panel]{background:#fcfcfc}[data-ff-dashboard-left-panel],[data-ff-dashboard-right-panel]{background:#fff}[data-ff-dashboard-left-panel]{border-right:2px solid #dbdbdb}[data-ff-dashboard-right-panel]{border-left:2px solid #dbdbdb}[data-ff-dashboard-panel-body]{overflow-x:auto}
```
Comment (jayrbolton) [inline public/client/css/dashboard/page.css]: Remove anything in `public/client` including the directory itself. It should be gitignore in .gitignore. you can `git rm x` to do it.

**Pull request merged**